.drone.yml

```diff
@@ -18,3 +15,15 @@ publish:
       - $(git rev-parse --short HEAD)
     when:
       branch: master
+services:
+  - postgres
+  - redis
+script:
+  - go get -u ./...
+  - sudo apt-get install -y postgresql-client
+  - createdb -h localhost -U postgres drone
+  - ls -alF
+  - pwd
+  - git log --stat
```

Comment:
That's actually for debugging due to drone being funky. The `integration_tests.sh` script is missing in drone (check the logs)